src/randoop/main/ClassTypeLoader.java

```diff
@@ -0,0 +1,134 @@
+package randoop.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.types.TypeNames;
+import randoop.util.Files;
+
+import plume.EntryReader;
+
+/**
+ * ClassTypeLoader provides methods that allow a list of class names to be loaded
+ * as {@link Class<?>} objects.
+ * 
+ * (Extracted from class previously known as randoop.util.Reflection.)
+ * 
+ * @author bjkeller
+ *
+ */
+public class ClassTypeLoader {
+
+  /**
+   * loadClassesFromList returns a list of {@link Class} objects given a list of class names.
+   * In the case that the typename recognition throws an exception, an error is only thrown if
+   * <code>noerr</code> is false.
+   * 
+   * @param classNames list of strings for classnames.
+   * @param noerr a boolean flag to indicate whether to report classes not found.
+   * @return list of {@link Class} objects corresponding to elements of list.
+   */
+  public static List<Class<?>> loadClassesFromList(List<String> classNames, boolean noerr) {
+    List<Class<?>> result = new ArrayList<Class<?>>(classNames.size());
+    for (String className : classNames) {
+      try {
+        Class<?> c = TypeNames.recognizeType(className);
```

Comment:
I mention this lower down, but it's confusing that this method is named "load...", it calls one named "recognize...", and that calls two named "forName".  Let's use consistent names for these throughout.  Ones with "forName" seem reasonable to me, especially since that is the name used in the JDK.